### PR TITLE
Refactored and finished eureka registration. Added logging

### PR DESCRIPTION
**Pull request by R4il**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,16 @@ A simple Node.js app using restify that interacts with the googleMaps directions
...
 
+This service currently provides the following endpoints:
+
+* POST to api/v1/diretions with a body :
+
+```js
+{
+	"origin": "string value",
+	"destination": "string value",
+	"departure_time": "time of departure in epoch"
```
Comment (Castyr) [inline README.md]: Don't put underscores in JSON keys, the cause issues with parsers like Jackson.

File: src/app.js [hunk 2]
```diff
@@ -1,54 +1,13 @@
...
-  console.log("Waiting for eureka.");
-  await sleep(25000);
-  eureka.start((error) => {
-    console.log(error || 'complete');
-  });
-}
-waitReady();
-
 app.listen(port, () => {
   console.log('%s listening at %s', app.name, app.url);
```
Comment (Castyr) [inline src/app.js]: This should be an info log from winston.

File: src/test/contract.test.js [hunk 3]
```diff
@@ -2,14 +2,14 @@ const path = require('path');
...
-    providerBaseUrl: 'http://localhost:8080',
-    pactUrls: [path.resolve(__dirname, '../../pacts/my_consumer-posts_provider.json')]
+  providerBaseUrl: 'http://localhost:8080',
+  pactUrls: [path.resolve(__dirname, '../../pacts/my_consumer-posts_provider.json')]
 };
 
 pact.verifyPacts(opts).then(() => {
-    console.log('success');
-    process.exit(0);
+  console.log('success');
```
Comment (Castyr) [inline src/test/contract.test.js]: These should also be winston logs.

**Review by Castyr — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (R4il) [reply README.md]: So, what is ideal here. camelCase?

**Review by idura007 — APPROVED** [review]

**Pull request merged**